tsconfig.json

```
@@ -29,7 +29,8 @@
     /* Strict Type-Checking Options */
     "strict": true,
     /* Enable all strict type-checking options. */
-    // "noImplicitAny": true,                 /* Raise error on expressions and declarations with an implied 'any' type. */
+    // "noImplicitAny": false,
```

```suggestion
    // "noImplicitAny": true,
```

`noImplicitAny` should be set as true. https://qiita.com/uhyo/items/aae57ba0734e36ee846a